test/e2e/facility-helpers.js

```diff
@@ -113,23 +112,23 @@ const resultsData = {
         },
         feedback: {
           health: {
-            effective_date_range: 'Jun 2016 - Nov 2016',
-            primary_care_routine: 0.73,
-            primary_care_urgent: 0.74,
-            specialty_care_routine: null,
-            specialty_care_urgent: null
+            effectiveDateRange: 'Sep 2016 - Feb 2017',
+            primaryCareRoutine: 0.78,
+            primaryCareUrgent: 0.73,
+            specialtyCareRoutine: null,
+            specialtyCareUrgent: null
           }
         },
         access: {
           health: {
-            primary_care_wait_days: 5.977272,
-            primary_care_wait_sample_size: 44,
-            specialty_care_wait_days: 4.327272,
-            specialty_care_wait_sample_size: 110,
-            mental_health_wait_days: 3.962962,
-            mental_health_wait_sample_size: 27,
-            urgent_consult_percentage: null,
-            urgent_consult_sample_size: null
+            primaryCareWaitDays: 5.977272,
```

Comment:
This test data needs updating based on new API format - removal of Days from key, change to individual specialty care categories.